backend/apps/volontulo/views/api.py

```diff
@@ -65,6 +69,62 @@ def login_view(request):
     )
 
 
+@api_view(['POST'])
+@authentication_classes((CsrfExemptSessionAuthentication,))
+@permission_classes((AllowAny,))
+def register_view(request):
+    """REST API register view."""
+    if not request.user.is_authenticated():
+        email = request.data.get('email')
+        password = request.data.get('password')
+
+        try:
+            user = User.objects.create_user(
+                username=email,
+                email=email,
+                password=password,
+                is_active=False
+            )
+
+            user.save()
+            profile = UserProfile(user=user)
+            ctx = {'token': profile.uuid, 'angular_root':
+                   settings.ANGULAR_ROOT}
+            profile.save()
+            send_mail(request, 'registration', [user.email], context=ctx)
+        except IntegrityError:
+            pass
+
+        return Response(
+            status=status.HTTP_201_CREATED,
```

Comment:
You might attach here some response body. I think the newly created serialized user might be a good idea.